### ISSUE TITLE
Add support for runtime package mapping

### ISSUE THREAD

**Issue by lukaseder** [jOOQ/jOOQ#5631]
Packages act as an intermediary qualifier for:

- Types (see #330)
- Procedures

It may make sense for some users to be able to map them as well, e.g. to have different package versions in parallel.

**Comment by lukaseder** [comment]
Duplicate of https://github.com/jOOQ/jOOQ/issues/5563
